dom-manipulation/main.js

```diff
@@ -0,0 +1,24 @@
+let clicks = 0;
+const $button = document.querySelector('.hot-button');
+const $clicked = document.querySelector('.click-count');
+
+function buttonClicked(event) {
+  clicks++;
+  $clicked.textContent = 'Clicks: ' + clicks;
+  console.log($clicked.textContent);
```

Comment:
Remove all console logs before submitting your exercise! Unless we specially ask for them in the instructions.